src/app/features/cart/cart.component.html

```diff
@@ -1,10 +1,14 @@
 <div class="cart">
+  <h2>{{ restaurantName }}</h2>
   <div class="cart-item" *ngFor="let item of items">
-    <span>{{ item.quantity }}</span> X
     <span>{{ item.name }}</span>
+    <span>{{ item.quantity }}</span> X
+    <span>{{ item.price / 100 }}$</span>
```

Comment:
Read about pipe and try to see if you can use the [CurrencyPipe](https://angular.io/api/common/CurrencyPipe) here and in src/app/shared/modal/modal.component.html

Reply:
I can't use it